doc/sphinx/source/howto/tune-for-deployment.rst

```diff
@@ -0,0 +1,171 @@
+.. tune-for-deployment.rst
+
+
+############################################
+Fine-tune code deployment to nGraph devices  
+############################################
+
+When deploying code to nGraph-enabled devices, there are some additional 
+configuration options that can be incorporated. Fine-tuning an nGraph-enabled
+device is as much of an art as it is a science; there are virtually limitless
+ways to command your training or inference computations to run the way you want.  
+
+Most of the time, and how this has worked :doc:`for many <../framework-integration-guides>` 
+of the "direct optimizations" we've shared with the developer community, 
+`engineering teams spend several weeks or months`_ working on extracting best 
```

Comment:
We dont want to create the impression that intel devices are hard to tune. "engineering teams carefully tune the workload to extract best performance..."

Reply:
Agree; changed.